sdk/client/task_resource.go

```diff
@@ -1394,6 +1397,10 @@ func (a *TaskResourceApiService) UpdateTask(ctx context.Context, taskResult *mod
 	return localVarReturnValue, localVarHttpResponse, nil
 }
 
+type TaskResourceApiUpdateTaskOpts struct {
+	Workerid optional.String
+}
```

Comment:
**suggestion (non-blocking):** Later (i.e. not as part of this change), we should look into replacing this pattern across the SDK with [functional options](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis).

Reply:
I ❤️ the functional option. Specifically for generated code - I'm not sure if we should custom it that much, because it may be trickier to update in the future. For the rest, we should do it ASAP!

Reply:
Agreed -- we'd be introducing a breaking change though. We should plan carefully and package them all up into a pill customers can swallow all at once. ;)